pages/components/PartnerCard/PartnerCard.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react'
+import { VerifiedPartner } from '../../interfaces/VerifiedPartner'
+import CategoryPill from '../CategoryPill/CategoryPill'
+import Discord from '../Icons/Discord'
+import Email from '../Icons/Email'
+import Instagram from '../Icons/Instagram'
+import Linkedin from '../Icons/Linkedin'
+import Twitter from '../Icons/Twitter'
+import Youtube from '../Icons/Youtube'
+import Website from '../Icons/Website'
+
+import styles from './PartnerCard.module.css'
+import Marketplace from '../Icons/Marketplace'
+import Opensea from '../Icons/Opensea'
+import Icon from './Icon'
+import { FormattedMessage } from 'react-intl'
+import DetailsList from './DetailsList'
+
+interface Props {
+  partner: VerifiedPartner
+}
+
+const DATA_URL = process.env.NEXT_PUBLIC_PARTNERS_DATA_URL
+
+function PartnerCard({ partner }: Props) {
+  const WEBSITE = partner.website || ''
+
+  return (
+    <div className={styles.PartnerCard}>
```

Comment:
CSS module are scoped to the module right? So maybe this "global/main" classes we can call them Container? Or something like that.

Reply:
Also we can write them in lowercase... so `styles.container`, `styles.image`. Somehow the first letter capitalized feels wrong in this context 😓